platform/pulpcore/app/models/content.py

```diff
@@ -43,6 +43,37 @@ def storage_path(self, name):
     sha384 = models.CharField(max_length=96, blank=False, null=False, unique=True, db_index=True)
     sha512 = models.CharField(max_length=128, blank=False, null=False, unique=True, db_index=True)
 
+    # All digest fields ordered by algorithm strength.
+    DIGEST_FIELDS = (
+        'sha512',
+        'sha384',
+        'sha256',
+        'sha224',
+        'sha1',
+        'md5',
+    )
+
+    # Reliable digest fields ordered by algorithm strength.
+    RELIABLE_DIGEST_FIELDS = DIGEST_FIELDS[:-1]
```

Comment:
@mhrivnak: Looks like we only have unique=True on sha256 field and higher.  The `RELIABLE_DIGEST_FIELDS` should probably match, right?